### DISCUSSION TITLE
Dropdown filter using special characters (accents, diacritics...): Some of them work, while others not

### DISCUSSION THREAD

**Discussion by persdv** [primefaces/community#1249] [PrimeNG] [answered]
Hello

I'm trying to use the dropdown filters (p-dropdown PrimeNG component), but I'm struggling when filtering text that contains diacritics.

If the dropdown contains "España" and I type "Espana" it works as I would expect, and it returns "España" (OK for me)
The same thing occurs for other characters like "á", "è", "Ý". I've noticed that for "Š" character, it didn't work in version 16, but it works in version 17 (OK for me).

However, if the dropdown contains "WŁOCHY" and I type "WLOCHY" the dropdown is empty (NOK for me).

Is there any way to make " **Ł** " character to work in the same way when using the filters?

Thank you in advance

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Filter template can take a callback in dropdown, you can use your callback to perform filtering see the example: https://stackblitz.com/run?file=src%2Fapp%2Fdemo%2Fdropdown-filter-demo.html,src%2Fapp%2Fdemo%2Fdropdown-filter-demo.ts

You probably need to update method bound to keyup and invoke filter after performing your custom logic.

**Reply by persdv** [reply]
Thank you very much. But I'm afraid the link doesn't work. It would be great if you had a working link.
